CHANGELOG.md

```diff
@@ -3,6 +3,55 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.7.0
+
+_Jun 6, 2024_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- ✨ Introduced plugins system in the Charts @alexfauquette
```

Comment:
@alexfauquette we probably want to leave this out for now? 😅

Reply:
out of the highlights? or generally out of the CHANGELOG?

Reply:
AFAIK it's purely an internal thing so our of the highlights

Reply:
And the release can probably be a patch, I guess this is the PR that justified the minor for you :+1: